style.css

```diff
@@ -87,3 +88,278 @@ body {
   justify-content: flex-start;
   padding-left: 5px;
 }
+
+.page1 {
+  background-color: var(--v3);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-repeat: no-repeat;
```

Comment:
- [ ] Please kindly use both the display technologies of CSS in this section. The section container which contains projects should be `display: Grid,` and the individual project `display: flex`

***Requirement for the Project***
![Screenshot from 2022-10-29 09-22-52](https://user-images.githubusercontent.com/65129228/198817397-db36becc-f4b6-4d58-8fd4-2396303c6e04.png)